lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Yunge Zhu, <yungez@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_keyvault_info
+version_added: "2.9"
+short_description: Get Azure Key Vault facts.
+description:
+    - Get facts of Azure Key Vault.
```

Comment:
I think we should have "Get info" considering renaming ;-) but that's minow

Reply:
i ever think about it. but i think `facts` here is more common used in ansible, so keep facts.

Reply:
it's ok, such things we can rename later anyway if we decide to